tests/integration/components/auto-complete-test.js

```diff
@@ -13,487 +15,494 @@ import {
   clickTrigger,
   currentOptions } from '../../helpers/ember-power-select';
 
-moduleForComponent('auto-complete', 'Integration | Component | {{auto-complete}}', {
-  integration: true,
+module('Integration | Component | {{auto-complete}}', function(hooks) {
+  setupRenderingTest(hooks);
```

Comment:
This new way of testing is also to get rid of jQuery. Now you didn't refactor it as much as it could/should. I still don't see why you needed to change al these tests...

For now I think you should rollback the changes or keep it this way for now, the rest of the refactoring will come later.